app/services/withdraw_participant.rb

```diff
@@ -36,8 +36,11 @@ def call
     unless participant_profile.npq?
       induction_coordinator = participant_profile.school.induction_coordinator_profiles.first
       if induction_coordinator.present?
-        SchoolMailer.with(withdrawn_participant: participant_profile, induction_coordinator:)
-                    .fip_provider_has_withdrawn_a_participant.deliver_later
+        SchoolMailer.with(
+          withdrawn_participant: participant_profile,
+          induction_coordinator:,
+          partnership: relevant_induction_record.partnership,
```

Comment:
Previously as it would return the first partnership that matched the school and cohort, it didn't account for challenged etc. But as we're passing in the partnership from the induction record, it should be good.

I'll speak to @cwrw to confirm

Reply:
yea can confirm that, if the relevant IR has the partnership that's the one we need and covers the challenged/relationship ones as well